local-cookbooks/cyclescape/templates/default/passenger.vhost.conf

```diff
@@ -80,14 +80,22 @@
       RewriteCond %{HTTP_HOST} ^www\.(.+)\.cyclescape.org$ [NC]
       RewriteRule ^/(.*) https://%1.cyclescape.org/$1 [L,R]
 
+      SSLCertificateFile /etc/apache2/ssl/cert.pem
+      SSLCertificateKeyFile /etc/apache2/ssl/privkey.pem
+      SSLCertificateChainFile /etc/apache2/ssl/chain.pem
```

Comment:
@mvl22 I've made these files symlinks to the existing certs so the change is backwards compatible.